.docker/ci/Dockerfile

```diff
@@ -1,48 +1,85 @@
-# moveit/moveit:melodic-ci
+# moveit/moveit2:crystal-ci
 # Sets up a base image to use for running Continuous Integration on Travis
+FROM ros:crystal-ros-base-bionic
 
-FROM ros:melodic-ros-base
-MAINTAINER Dave Coleman dave@picknik.ai
+LABEL maintainer="mike@picknik.ai"
 
 ENV TERM xterm
 
 # Setup catkin workspace
-ENV CATKIN_WS=/root/ws_moveit
-WORKDIR $CATKIN_WS
+ENV ROS_WS=/root/ws_moveit
+ENV ROS_DISTRO crystal
+
+WORKDIR $ROS_WS
 
 # Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
 # https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
-RUN \
-    mkdir src && \
-    cd src && \
+
+RUN mkdir src && \
```

Comment:
I 100% agree. I have been very frustrated by this while developing locally as well. I believe the rational behind a single run command is to minimize the number of layers. See the comment above:

```
# Commands are combined in single RUN statement with "apt/lists" folder removal to reduce image size
# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
```

Reply:
This decision predates my involvement with MoveIt. I am merely maintaining the existing style

Reply:
>  single RUN statement with "apt/lists" folder removal to reduce image size

That old comment is misleading. We just need to make sure our RUN commands clean up after themselves before writing the next layer, that all. We should not be beholden to monolithic RUN commands to avoid this having to tidy up, that's almost an anti pattern. Best remedy it now while we have the hammer out.

I'd break up the apt install, the pip install, and the rosdep install into separate RUN commands. Just be sure to `apt update` and `rm -rf /var/lib/apt/lists/*` at the start and end of the both apt and rosdep install steps.